### PR TITLE
Add install info to GIS applications

### PR DESCRIPTION
**Pull request by samumantha**
Installation info added to almost all GIS applications.

SAGA in r-env, link to be added if r-env gets install info.

please **squash** and merge

### CODE REVIEW

File: docs/apps/ames-stereo.md [hunk 1]
```diff
@@ -65,6 +65,10 @@ NASA Ames Stereo Pipeline (ASP) is distributed under [Apache-2 license](https://
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports. As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
+## Installation
+
+Ames Stereo Pipeline was installed on Puhti using [Tykkys conda-containerize functionality](../computing/containers/tykky.md). The package `stereo-pipeline` is available from conda channels: nasa-ames-stereo-pipelin, usgs-astrogeology and conda-forge (all three must be added to channels!).
```
Comment (ktiits) [inline docs/apps/ames-stereo.md:70]:
```suggestion
Ames Stereo Pipeline was installed on Puhti using [Tykky's conda-containerize functionality](../computing/containers/tykky.md), using this [Ames Stereo Pipeline conda environment file](https://github.com/csc-training/geocomputing/blob/master/supercomputer_installations/ames-stereo_3.2.0.yml).
```

File: docs/apps/cloudcompare.md [hunk 2]
```diff
@@ -43,18 +43,21 @@ CloudCompare is published under the [GNU General Public License](https://github.
...
 
 If you used a [CloudCompare plugin](http://www.cloudcompare.org/doc/wiki/index.php?title=Plugins), also cite the plugin authors.
 
 ## Acknowledgement
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports. As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
+## Installation
+
+Cloudcompare was installed to Puhti via Singularity. The Singularity definition file is available on [CSCs 'singularity-recipes' repository](https://github.com/CSCfi/singularity-recipes/blob/main/cloudcompare/cloudcompare.def).
```
Comment (ktiits) [inline docs/apps/cloudcompare.md:60]:
```suggestion
CloudCompare was installed to Puhti with Apptainer, using this [CloudCompare Apptainer definition file](https://github.com/CSCfi/singularity-recipes/blob/main/cloudcompare/cloudcompare.def).
```

File: docs/apps/force.md [hunk 3]
```diff
@@ -56,6 +56,10 @@ This software can be cited by citing the appropriate [FORCE publication](https:/
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
+## Installation
+
+FORCE was installed on Puhti with Singularity using the [Docker image provided by the FORCE author on Dockerhub](https://hub.docker.com/r/davidfrantz/force). The container was then wrapped with [Tykky](../computing/containers/tykky.md).
```
Comment (ktiits) [inline docs/apps/force.md:61]:
```suggestion
FORCE was installed to Puhti with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [Docker image provided by the FORCE author on Dockerhub](https://hub.docker.com/r/davidfrantz/force). 

The container was then wrapped 
```

File: docs/apps/gdal.md [hunk 4]
```diff
@@ -98,12 +98,19 @@ Open Source Geospatial Foundation.
...
+
 ## Acknowledgement
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation 
+
+Standalone GDAL was installed to Puhti using [Spack](https://spack.io/). For other installations, see respective application page.
```
Comment (ktiits) [inline docs/apps/gdal.md:112]:
```suggestion
Standalone GDAL was installed to Puhti using [Spack and its GDAL package definition](https://spack.readthedocs.io/en/latest/package_list.html#gdal). For other installations, see respective application page.
```

File: docs/apps/geoconda.md [hunk 5]
```diff
@@ -138,6 +138,10 @@ Please see the above linked package pages for citation information per package.
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
+## Installation
+
+Geoconda was installed on Puhti using [Tykkys conda-containerize functionality](../computing/containers/tykky.md). The environment file is available in [CSCs geocomputing repository](https://raw.githubusercontent.com/csc-training/geocomputing/master/puhti_geoconda_environment.yml). Note that for reproducibility, you'll need to define the package versions in the environment file, which can be checked on Puhti using `list-packages` command after loading the `geoconda` module.
```
Comment (ktiits) [inline docs/apps/geoconda.md:143]:
```suggestion
Geoconda was installed to Puhti and Mahti using [Tykkys conda-containerize functionality](../computing/containers/tykky.md). Geoconda conda environment files are available in [CSCs geocomputing repository](https://github.com/csc-training/geocomputing/tree/master/supercomputer_installations). Note that for reproducibility, you'll need to define the package versions in the environment file, which can be checked on Puhti and Mahti using `list-packages` command after loading the `geoconda` module.
```

File: docs/apps/grass.md [hunk 6]
```diff
@@ -52,16 +52,23 @@ grass
...
+
 ## Acknowledgement
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+GRASS was installed on Puhti together with QGIS with Singularity using the [Docker image provided by QGIS on Dockerhub](https://hub.docker.com/r/qgis/qgis). The container was then wrapped with [Tykky](../computing/containers/tykky.md).
```
Comment (ktiits) [inline docs/apps/grass.md:69]:
```suggestion
GRASS was installed to Puhti and LUMI as part of [QGIS installation](qgis.md#installation).
```

File: docs/apps/lastools.md [hunk 7]
```diff
@@ -122,6 +123,10 @@ Citation of the software depends on which license was used:
 * M. Isenburg, "LAStools - efficient LiDAR processing software" (version 220613, unlicensed), obtained from http://rapidlasso.com/LAStools
 * rapidlasso GmbH, "LAStools - efficient LiDAR processing software" (version 220613, commercial), obtained from http://rapidlasso.com/LAStools
 
+## Installation
+
+Lastools was installed on Puhti with Singularity using the [Docker image from Dockerhub](https://hub.docker.com/r/pydo/lastools). The container was then wrapped with [Tykky](../computing/containers/tykky.md).
```
Comment (ktiits) [inline docs/apps/lastools.md:128]:
```suggestion
Lastools was installed on Puhti with Singularity using the [Docker image from Dockerhub](https://hub.docker.com/r/pydo/lastools). The container was then wrapped with [Tykky](../computing/containers/tykky.md).

LasTools was installed to Puhti with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [LasTools Docker image from Dockerhub](https://hub.docker.com/r/pydo/lastools). 
```

File: docs/apps/opendronemap.md [hunk 8]
```diff
@@ -78,12 +78,20 @@ OpenDroneMap is distributed under the GNU General Public License (GPL) version 3
...
+
 ## Acknowledgement
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+Opendronemap was installed on Puhti with Singularity using the [Docker image provided by Opendronemap](https://hub.docker.com/r/opendronemap/odm). The container was then wrapped with [Tykky](../computing/containers/tykky.md).
```
Comment (ktiits) [inline docs/apps/opendronemap.md:92]:
```suggestion
OpenDroneMap was installed to Puhti with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [OpenDroneMap Docker image from Dockerhub provided by OpenDroneMap community](https://hub.docker.com/r/opendronemap/odm). 
```

File: docs/apps/otb.md [hunk 9]
```diff
@@ -89,12 +89,20 @@ Orfeo ToolBox is licensed under the Apache License, Version 2.0. [The full Orfeo
...
+
 ## Acknowledgement
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+OTB was installed on Puhti with Singularity using the [Docker image provided by OTB](https://hub.docker.com/r/orfeotoolbox/otb). The container was then wrapped with [Tykky](../computing/containers/tykky.md).
```
Comment (ktiits) [inline docs/apps/otb.md:103]:
```suggestion
Orfeo Toolbox was installed to Puhti with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [Orfeo Toolbox Docker image from Dockerhub provided by OTB community](https://hub.docker.com/r/orfeotoolbox/otb). 
```

File: docs/apps/pcl.md [hunk 10]
```diff
@@ -41,16 +41,22 @@ Note, that PCL/1.9.1 includes less PCL modules.
...
+
 ## Acknowledgement
 
 Please acknowdlege CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+PCL was installed to Puhti via Singularity. The Singularity definition file is available on [CSCs 'singularity-recipes' repository](https://github.com/CSCfi/singularity-recipes/blob/main/pcl/pcl_from_ubuntu.def).
```
Comment (ktiits) [inline docs/apps/pcl.md:58]:
```suggestion
PCL was installed to Puhti with Apptainer, using this [PCL Apptainer definition file](https://github.com/CSCfi/singularity-recipes/blob/main/pcl/pcl_from_ubuntu.def).
```

File: docs/apps/qgis.md [hunk 11]
```diff
@@ -65,12 +65,20 @@ QGIS is licensed under the GNU General Public License.
...
+
 ## Acknowledgement
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+QGIS was installed on Puhti with Singularity using the [Docker image provided by QGIS on Dockerhub](https://hub.docker.com/r/qgis/qgis). The container was then wrapped with [Tykky](../computing/containers/tykky.md).
```
Comment (ktiits) [inline docs/apps/qgis.md:79]:
```suggestion
QGIS was installed to Puhti and LUMI with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [QGIS Docker image from Dockerhub provided by QGIS community](https://hub.docker.com/r/qgis/qgis). 
```

File: docs/apps/snap.md [hunk 12]
```diff
@@ -159,12 +159,20 @@ All SNAP software is published under the [GPL-3](https://www.gnu.org/licenses/gp
...
+
 ##  Acknowledgement
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+SNAP was installed on Puhti with Singularity using the [Docker image provided by mundialis on Dockerhub](https://hub.docker.com/r/mundialis/esa-snap) with some small additions to provide snappy and snapista Python interfaces. The Singularity definition file can be found in [CSCs singularity-recipes reposiory](https://raw.githubusercontent.com/CSCfi/singularity-recipes/main/snap/snap_py.def). The container was then wrapped with [Tykky](../computing/containers/tykky.md).
```
Comment (ktiits) [inline docs/apps/snap.md:173]:
```suggestion
SNAP was installed on Puhti with Singularity using the [SNAP Docker image provided by mundialis on Dockerhub](https://hub.docker.com/r/mundialis/esa-snap) with some small additions to provide snappy and snapista Python interfaces. The container was finally wrapped with [Tykky's wrap-container functionality](../computing/containers/tykky.md). The full [SNAP Singularity definition file](https://raw.githubusercontent.com/CSCfi/singularity-recipes/main/snap/snap_py.def). 
```

File: docs/apps/whiteboxtools.md [hunk 13]
```diff
@@ -60,12 +60,20 @@ The WhiteboxTools open-core platform is distributed under the MIT license. [The
...
+
 ## Acknowledgement
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+Please see the [Geoconda page](./geoconda.md).
```
Comment (ktiits) [inline docs/apps/whiteboxtools.md:74]:
```suggestion
Whiteboxtools are part of [Geoconda installation](./geoconda.md#installation).
```

File: docs/apps/zonation.md [hunk 14]
```diff
@@ -68,16 +68,23 @@ srun z5 -w --mode=ABF minimal_settings.z5 <path_to_where_you_want_to_store_the_r
...
+
 ## Acknowledgement
 
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+Zonation was installed to Puhti via Singularity. The Singularity definition file is available on [CSCs 'singularity-recipes' repository](https://raw.githubusercontent.com/CSCfi/singularity-recipes/main/zonation/zonation5.def) .
```
Comment (ktiits) [inline docs/apps/zonation.md:85]:
```suggestion
Zonation was installed on Puhti with Apptainer using [Zonation Apptainer definition file written by CSC](https://raw.githubusercontent.com/CSCfi/singularity-recipes/main/zonation/zonation5.def). The container was finally wrapped with [Tykky's wrap-container functionality](../computing/containers/tykky.md). 
```

**Comment by samumantha** [comment]
For prefix I would not put /appl/soft/... path anywhere. Users cannot install there. Neither just "versionnumber" or such but rather /path/to/where/you/want/to/install/the/tool or something like this, so that they understand what that path is?

**Comment by samumantha** [comment]
The yml files could maybe also rather be collected in the singularity recipe repository than geocomputing?

**Comment by ktiits** [comment]
> For prefix I would not put /appl/soft/... path anywhere. Users cannot install there. Neither just "versionnumber" or such but rather /path/to/where/you/want/to/install/the/tool or something like this, so that they understand what that path is?

Removed all /appl/soft beginnings and prefix after to "install_dir".

**Comment by ktiits** [comment]
> The yml files could maybe also rather be collected in the singularity recipe repository than geocomputing?

I was wondering which to choose and had no clear preference. You had the geoconda one in Geocomputing, so added there. The other is more common, so somebody might not like that we "pollute" it with other stuff..

**Comment by samumantha** [comment]
Ok, let's leave it there for now. I will add it to topics to be discussed in bigger round later.

**Comment by samumantha** [comment]
all committed, ready for review

File: docs/apps/gdal.md [hunk 15]
```diff
@@ -98,12 +98,19 @@ Open Source Geospatial Foundation.
 URL https://gdal.org, 
 DOI: 10.5281/zenodo.5884351
 
+## Acknowledgement
```
Comment (ktiits) [inline docs/apps/gdal.md:101]:
```suggestion

```

File: docs/apps/opendronemap.md [hunk 16]
```diff
@@ -78,12 +78,20 @@ OpenDroneMap is distributed under the GNU General Public License (GPL) version 3
 
 `OpenDroneMap Authors, ODM - A command line toolkit to generate maps, point clouds, 3D models and DEMs from drone, balloon or kite images. OpenDroneMap/ODM GitHub Page 2020; https://github.com/OpenDroneMap/ODM`
 
+## Acknowledgement
```
Comment (ktiits) [inline docs/apps/opendronemap.md:81]:
```suggestion
```

File: docs/apps/otb.md [hunk 17]
```diff
@@ -89,12 +89,22 @@ Orfeo ToolBox is licensed under the Apache License, Version 2.0. [The full Orfeo
 
 `Grizonnet, M., Michel, J., Poughon, V. et al. Orfeo ToolBox: open source processing of remote sensing images. Open geospatial data, softw. stand. 2, 15 (2017). https://doi.org/10.1186/s40965-017-0031-6`
 
+## Acknowledgement
```
Comment (ktiits) [inline docs/apps/otb.md:92]:
```suggestion
```

File: docs/apps/pdal.md [hunk 18]
```diff
@@ -34,12 +34,20 @@ PDAL is licensed with the BSD open source license. [The full PDAL licence](https
 
 ```  PDAL contributors (2022). PDAL: The Point Data Abstraction Library, DOI: 10.5281/zenodo.2616780  ```
 
+## Acknowledgement
```
Comment (ktiits) [inline docs/apps/pdal.md:37]:
```suggestion
```

File: docs/apps/qgis.md [hunk 19]
```diff
@@ -65,12 +65,22 @@ QGIS is licensed under the GNU General Public License.
 
 ```QGIS.org, 2023. QGIS Geographic Information System. QGIS Association. http://www.qgis.org```
 
+## Acknowledgement
```
Comment (ktiits) [inline docs/apps/qgis.md:68]:
```suggestion
```

File: docs/apps/snap.md [hunk 20]
```diff
@@ -159,12 +159,22 @@ All SNAP software is published under the [GPL-3](https://www.gnu.org/licenses/gp
...
 
 
+## Installation
+
+SNAP was installed to Puhti with Singularity using the [SNAP Docker image provided by mundialis on Dockerhub](https://hub.docker.com/r/mundialis/esa-snap) with some small additions to provide snappy and snapista Python interfaces. The container was finally wrapped with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations): `wrap-container -w /usr/local/snap/bin,/usr/bin snap9_py.sif --prefix install_dir`
+
+The full [SNAP Singularity definition file](https://raw.githubusercontent.com/CSCfi/singularity-recipes/main/snap/snap_py.def). 
+
+
 ### References
```
Comment (ktiits) [inline docs/apps/snap.md:178]:
```suggestion
## References
```

File: docs/apps/sen2cor.md [hunk 21]
```diff
@@ -40,16 +40,23 @@ Here you can adjust it to your needs using any editor and add it to the call: `L
...
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+Sen2cor was installed on Puhti using [ESAs standalone Sen2cor installer for Linux](https://step.esa.int/main/snap-supported-plugins/sen2cor/).
+
+
 ### References
```
Comment (ktiits) [inline docs/apps/sen2cor.md:60]:
```suggestion
## References
```

File: docs/apps/qgis.md [hunk 22]
```diff
@@ -65,12 +65,22 @@ QGIS is licensed under the GNU General Public License.
...
 
 
+## Installation
+
+QGIS was installed to Puhti and LUMI with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [QGIS Docker image from Dockerhub provided by QGIS community](https://hub.docker.com/r/qgis/qgis). In LUMI `Tykky` is named `Container wrapper`.
+
+`wrap-container -w /usr/bin docker://qgis/qgis:latest --prefix install_dir`
+
+
 ### References
```
Comment (ktiits) [inline docs/apps/qgis.md:84]:
```suggestion
## References
```

File: docs/apps/pdal.md [hunk 23]
```diff
@@ -34,12 +34,20 @@ PDAL is licensed with the BSD open source license. [The full PDAL licence](https
...
 Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports.
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
 
+## Installation
+
+Please see the respectives versions module page: [Geoconda](./geoconda.md) , [QGIS](./qgis.md).
+
+
 ### References
```
Comment (ktiits) [inline docs/apps/pdal.md:51]:
```suggestion
## References
```

File: docs/apps/otb.md [hunk 24]
```diff
@@ -89,12 +89,22 @@ Orfeo ToolBox is licensed under the Apache License, Version 2.0. [The full Orfeo
...
 
 
+## Installation
+
+Orfeo Toolbox was installed to Puhti with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [Orfeo Toolbox Docker image from Dockerhub provided by OTB community](https://hub.docker.com/r/orfeotoolbox/otb). 
+
+`wrap-container -w /opt/otb/bin,/usr/bin/python3 docker://orfeotoolbox/otb:8.0.1 --prefix install_dir`
+
+
 ### References
```
Comment (ktiits) [inline docs/apps/otb.md:108]:
```suggestion
## References
```

File: docs/apps/lastools.md [hunk 25]
```diff
@@ -122,6 +123,12 @@ Citation of the software depends on which license was used:
...
 * rapidlasso GmbH, "LAStools - efficient LiDAR processing software" (version 220613, commercial), obtained from http://rapidlasso.com/LAStools
 
+## Installation
+
+LAStools was installed to Puhti with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [LAStools Docker image from Dockerhub](https://hub.docker.com/r/pydo/lastools). 
+
+`wrap-container -w /opt/LAStools docker//:pydo/lastools:latest --prefix install_dir`
+
 
 ### References
```
Comment (ktiits) [inline docs/apps/lastools.md:133]:
```suggestion
## References
```

File: docs/apps/force.md [hunk 26]
```diff
@@ -56,6 +56,12 @@ This software can be cited by citing the appropriate [FORCE publication](https:/
...
 As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".
 
+## Installation
+
+FORCE was installed to Puhti with [Tykky's wrap-container functionality](../computing/containers/tykky.md#container-based-installations) using the [Docker image provided by the FORCE author on Dockerhub](https://hub.docker.com/r/davidfrantz/force). 
+
+`wrap-container -w /home/docker/bin docker://davidfrantz/force:latest --prefix install_dir`
+
 
 ### References
```
Comment (ktiits) [inline docs/apps/force.md:66]:
```suggestion
## References
```

**Pull request merged**